sls-public.html

```diff
@@ -0,0 +1,370 @@
+---
+layout: sls
+title: Publishing Data using Nightshade
+pdfs:
+- sls-public.pdf
+description: This page demonstrates the use of the School of Life Sciences (SLS) OMERO server to make images on the server viewable by people who are not registered users, for example on a web page.
+redirect_from:
+  - /cls-public-demo.html
```

Comment:
As the `sls-public-demo` file has been removed as part of f18740acd562253e6c15b911a1f19efcf12534a9, can it be added to this list so that it redirects to the new permanent URL for the data publishing?